### ISSUE TITLE
Deploy 1.4.0-rc.2

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/resistance-in-a-wire#117]
To track what needs to be done for 1.4.0-rc.2. See https://github.com/phetsims/QA/issues/66 for previous RC.

- [x]  verify https://github.com/phetsims/john-travoltage/issues/252 is fixed in this version
- [x] cherry pick commits for help dialog updates (https://github.com/phetsims/a11y-research/issues/62)
  - https://github.com/phetsims/scenery-phet/commit/24bd572b2c0068b2569254b6730a6d69d64fb807
  - https://github.com/phetsims/scenery-phet/commit/b416cb4c16b816a59ea1bb58844fa7f2b53df152
  - https://github.com/phetsims/resistance-in-a-wire/commit/e46c2a4905ef29bd7f6587c67d80a3037294df34
- [x] Check status of build process to make sure we are clear for `deploy-rc`.

**Comment by jessegreenberg** [comment]
> Check status of build process to make sure we are clear for deploy-rc.

On slack, got verification that we should be OK. @jonathanolson said
>Should be fine to use the normal checklist for things tonight, I don't believe anything should be broken or significantly different
> And let me know if you run into any issues. I think @mattpen upgraded some handling for how build-server requests get made

**Comment by jessegreenberg** [comment]
> cherry pick commits for help dialog updates (phetsims/a11y-research#62)

I would like to get review comments from a few more assignees before proceeding.

**Comment by jessegreenberg** [comment]
We got all  the 👍 we need in https://github.com/phetsims/a11y-research/issues/62,  I think we are ready to go!

**Comment by jessegreenberg** [comment]
I tried to cherry-pick commits  in for dialog changes but ran into too many merge conflicts, will have to grab a few more in between the ones listed in https://github.com/phetsims/resistance-in-a-wire/issues/117#issue-275890400.

**Comment by jessegreenberg** [comment]
Done, https://github.com/phetsims/resistance-in-a-wire/commit/e46c2a4905ef29bd7f6587c67d80a3037294df34 isn't necessary on the release branch.

**Comment by jessegreenberg** [comment]
We are ready for a spot check RC.

**Comment by jessegreenberg** [comment]
RC was deployed and verification issue opened, closing this issue.
